Java/src/main/java/airport/Airport.java

```diff
@@ -1,8 +1,10 @@
-import Planes.experimentalPlane;
+package airport;
+
+import planes.ExperimentalPlane;
 import models.MilitaryType;
-import Planes.MilitaryPlane;
-import Planes.PassengerPlane;
-import Planes.Plane;
+import planes.MilitaryPlane;
+import planes.PassengerPlane;
+import planes.Plane;
 
 import java.util.*;
```

Comment:
Please review unused enum values (and the whole enum classes) in models package and remove them

Reply:
Looks like all used
![image](https://user-images.githubusercontent.com/10107317/183021205-e6550be4-d841-4b84-a9f9-506f58f96d76.png)

Reply:
I meant enums in "models" package:
<img width="670" alt="image" src="https://user-images.githubusercontent.com/90613721/183056375-c44559aa-d37a-49aa-92a1-2a7c2ef5314d.png">
<img width="703" alt="image" src="https://user-images.githubusercontent.com/90613721/183056404-d5dbd3f9-90d6-45b1-9c30-7ec8d65ef5fd.png">
however, we've discussed that these changes are optional - up to you if you decide to leave all or that are used in the code only

Reply:
Deleted unused enum values